rfq/negotiator.go

```diff
@@ -541,6 +541,43 @@ func (n *Negotiator) HasAssetSellOffer(assetID *asset.ID,
 	return true
 }
 
+// BuyOffer is a struct that represents an asset buy offer. This data structure
+// describes the maximum amount of an asset that this node is willing to
+// purchase.
+//
+// A buy offer is passive (unlike a buy order), meaning that it does not
+// actively lead to a buy request being sent to a peer. Instead, it is used by
+// the node to selectively accept or reject incoming asset sell quote requests
+// before price is considered.
+type BuyOffer struct {
+	// AssetID represents the identifier of the subject asset.
+	AssetID *asset.ID
+
+	// AssetGroupKey is the public group key of the subject asset.
+	AssetGroupKey *btcec.PublicKey
+
+	// MaxUnits is the maximum amount of the asset which this node is
+	// willing to purchase.
+	MaxUnits uint64
+}
```

Comment:
Doesn't the buy offer include any info for price per unit? Just the unit amount?

Reply:
Just trying to capture the very simplest case for now. As far as I understand, we don't want to extend the RFQ service too far beyond a MVP right now.